### PR TITLE
 Update file reader example

### PR DESCRIPTION
**Pull request by edgarriba**
depends on https://github.com/farm-ng/farm-ng-core/pull/47

### CODE REVIEW

File: py/examples/file_reader/main.py [hunk 1]
```diff
@@ -28,25 +28,18 @@ def main(file_name: str) -> None:
     reader = EventsFileReader(Path(file_name))
     assert reader.open()
 
-    # get a list with all the existing uri
-    uris: List[uri_pb2.Uri] = reader.uris()
+    # main window to visualize image
+    uris: List[uri_pb2.Uri] = reader.get_uris()
 
     # choose the Uri stream to seek in file
     uri: uri_pb2.Uri = uris[1]
```
Comment (Hackerman342) [inline py/examples/file_reader/main.py:35]: I think you'd talked of removing this hard coded value

**Review by Hackerman342 — APPROVED** [review]
LGTM

File: py/examples/file_reader/main.py [hunk 2]
```diff
@@ -28,25 +28,18 @@ def main(file_name: str) -> None:
...
-        event: event_pb2.Event
-        offset: int
-        event, offset = reader.get_event(uri, current_event)
+    events: List[EventLogPosition] = reader.get_events(uri)
 
+    for event_log in events:
         # seek and parse the message
         sample: oak_pb2.OakDataSample
-        sample = reader.read_message(event, offset)
+        sample = reader.read_message(event_log)
```
Comment (ethanrublee) [inline py/examples/file_reader/main.py:42]: In c++ the event_log_pos wrapper has a read function which allows for a little less boilerplate here, especially when reading multiple event logs simultaneously

**Review by ethanrublee — CHANGES_REQUESTED** [review]
The hard coded value in Uris is difficult to debug, just ran into it last night. Would be best to fix.

**Review by ethanrublee — APPROVED** [review]

**Pull request merged**